.appveyor.yml

```diff
@@ -1,41 +1,43 @@
 init:
- # -ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
- 
+# -ps: iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/appveyor/ci/master/scripts/enable-rdp.ps1'))
+
+version: 1.0.0.{build}
+
 configuration:
   - Release
   - Debug
 
 image:
   - Visual Studio 2015
-  - Visual Studio 2013
```

Comment:
Deleted Visual Studio 2013. Do we need it?

Reply:
I'm not sure if anywhere there is a stated list of expected supported versions. It's a bit of a bigger question - and certainly just having VS2013 and VS2015 probably does not actually cover enough things if this is the intent. If so - why VS2013 and not 2010 or 2012? Why not also 2017 and 2019?

Reply:
Initially committed by @russelltg. @russelltg do you still need it?

Reply:
i'm going to ditch the removal of VS2013 from this PR and issue in a dedicated PR - it can focus any debate about what is needed more clearly then, and a wider group can see

Reply:
Added VS2013 back - was a bit more of a pain than I expected, since i needed to select an appropriate pthreads version now, so there are a few more changes. It's done now - although I'd like to review needing it because it slows the compilation down a lot...